pythonx/async_clj_omni/fireplace.py

```diff
@@ -8,8 +9,15 @@ class Error(Exception):
 def gather_conn_info(vim):
     try:
         client = vim.eval("fireplace#client()")
-    except Exception:
-        raise Error
+    # If Fireplace complains that there is no REPL connection, silently fail
+    # instead of throwing an error in the user's face.
+    except NvimError as e:
+        if str(e) == "b'Fireplace: :Connect to a REPL or install classpath.vim'":
```

Comment:
This byte string -> string conversion ended up being necessary because byte strings are not equal to (byte) strings with the same value in Python 3.